config/default.json

```diff
@@ -0,0 +1,19 @@
+{
+    "development" : {
+        "postgres" : {
+            "database": "",
+            "username": "",
+            "password": ""
+        },
+        "email": {
+            "auth": {
+                "user":"itsupport@evgo.com",
+                "password":"EVgo2017!"
+            }
+        },
+        "logger": {
+            "filePath": "/Users/kjeter/Sites/express-boilerplate/utils/logs/log.txt",
```

Comment:
This shouldn't be an env setting and should output to a `/logs` folder in the project directory that is .gitignored.